test/verify/check-system-services

```diff
@@ -45,6 +45,7 @@ class TestServices(MachineCase):
         user_post_restore = "su - admin -c 'export XDG_RUNTIME_DIR=/run/user/$(id -u admin); systemctl --user daemon-reload; systemctl --user reset-failed'"
         self.restore_dir("/etc/systemd/user", user_post_restore, True)
         self.restore_dir("/etc/xdg/systemd/user", user_post_restore, True)
+        self.restore_dir("/home/admin/.config/systemd/user", user_post_restore, True)
```

Comment:
Mismatch with make_test_service below, which creates the file in /etc/systemd/user/. I think it's better to create it in ~admin, as it's more realistic, but the test needs to pick one consistently.

Reply:
Ah, this is for re-dropping the enablement symlink, as the test does not explicitly disable the service. I made it a bit more generic by restoring all of ~admin/.config.